### ISSUE TITLE
Current Global Settings Design Issue

### ISSUE THREAD

**Issue by saichaitanyaeruvaram** [GanuM/profitGuruPOS#549]
If user edits the global configuration, it effects all the data (ex items) which refers to the config(discount/tax).

User should be made aware of or think of consequences.

**Comment by moghribalakrishna** [comment]
absolutely.. we should put the fact that our user is new to software usage on top and avoid them committing innocent looking disasters.
